web/sites/default/settings.platformsh.php

```diff
@@ -6,22 +6,24 @@
 
 $platformsh = new \Platformsh\ConfigReader\Config();
 
+if($platformsh->hasRelationship('database')) {
+  // Configure the database.
+  $creds = $platformsh->credentials('database');
+  $databases['default']['default'] = [
+    'driver' => $creds['scheme'],
+    'database' => $creds['path'],
+    'username' => $creds['username'],
+    'password' => $creds['password'],
+    'host' => $creds['host'],
+    'port' => $creds['port'],
+    'pdo' => [PDO::MYSQL_ATTR_COMPRESS => !empty($creds['query']['compression'])]
+  ];
+}
+
 if (!$platformsh->inRuntime()) {
```

Comment:
I'm not sure about change this order. The if checks if the code is on Platform.sh if true load the database configuration otherwise returns does not load it. What you are doing is removing this condition.